sentry-core/src/main/java/io/sentry/core/Hub.java

```diff
@@ -1,13 +1,34 @@
 package io.sentry.core;
 
 import io.sentry.core.protocol.SentryId;
+import java.util.Deque;
+import java.util.concurrent.ConcurrentLinkedDeque;
 
 public class Hub implements IHub {
+
+  private class StackItem {
```

Comment:
Should be `static final` IMHO to not needlessly store the reference to the hub internally and as a slight hint to the jdk that it can potentially do more magic with it, respectively.